ui/admin/tests/e2e/README.md

```diff
@@ -35,10 +35,11 @@ You will need [Homebrew](https://brew.sh/) install. For secure persisting your S
 
 ### Accesses
 
-If you are missing any acccess, requested [through the IT service catalog](https://hashicorp.freshservice.com/support/catalog/items).
-
 - Doormat account: `boundary_team_acctest_dev`.
+  - Request access through Doormat.
 - HCP account: you will need to use Terraform cloud to spin up enos.
+- app.terraform.io org: `hashicorp-qti`.
+  - Request access in `#team-quality` slack channel.
```

Comment:
I think using #proj-boundary-qe might be a better option, they have the permissions as well and more context within the Boundary team.